MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -1464,4 +1464,13 @@ the Select` button. When clicked it closes all the previously selected tabs -->
 
     <!-- This string is displayed in the disable button of the quick Tracking Protection dialog, accessed from the Tracking Protection icon in the URL bar. -->
     <string name="tracking_dialog_button_disable">Disable</string>
+
+    <!-- This string is displayed in the body of the confirm dialog shown in some platforms when the back button is pressed to quit the app.  -->
```

Comment:
@keianhzo
Can you add to the comment what replaces the variable in the message?

Reply:
Added